backend/.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
+   "printWidth": 100,
+   "singleQuote": true,
+   "trailingComma": "es5",
+   "tabWidth": 3,
```

Comment:
had an interesting discussion with other collaborators about this but I think we'd like to keep this to 4 for now as a compromise on our preferences.

Reply:
ahh, will fix it to 4 tabs then asap. sorry and thank you for pointing it out!.